.github/workflows/connector-ci-tests.yml

```diff
@@ -90,10 +93,7 @@ jobs:
         if: matrix.connector
         uses: actions/checkout@v4
         with:
-          # In order to work correctly on forks, `repository` must be set if `ref` is set:
           repository: ${{ inputs.repo || github.event.pull_request.head.repo.full_name }}
-          # Java `integrationTestJava` Gradle task still calls Airbyte-CI, which fails on a detached head.
```

Comment:
Removed this comment. It is less relevant now, since all execution paths are now passing `repository` and `ref`.